.github/workflows/gem_notifications.yml

```diff
@@ -0,0 +1,38 @@
+name: Gem Notifications
+
+on:
+  schedule:
+    - cron:  '0 10 * * *'
+
+jobs:
+  gem_notifications:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: ruby/setup-ruby@v1.90.0
+        with:
+          ruby-version: 2.7
+      - uses: actions/checkout@v2
+      - run: gem install httparty
+      - name: Check for outdated gems 
+        run: ruby .github/workflows/scripts/slack_gem_notifications/notifications_script.rb ${{ env.gems }}
```

Comment:
^-- `notifications_script.rb` still references `ci_cron.yml`

Reply:
Thank you and done!

Reply:
Resolved with 11076e8